features/lwipstack/LWIPStack.h

```diff
@@ -450,7 +450,7 @@ class LWIP : public OnboardNetworkStack, private mbed::NonCopyable<LWIP> {
      *  @param func     Callback to be called
      *  @return         0 on success, negative error code on failure
      */
-    nsapi_error_t call_in(int delay, mbed::Callback<void()> func);
+    virtual nsapi_error_t call_in(int delay, mbed::Callback<void()> func);
```

Comment:
What was the reasoning for making this virtual? Just curious, didn't see this change referenced elsewhere in the PR.

Reply:
Resolved warning https://github.com/ARMmbed/mbed-os/pull/8592/commits/022ebd14e28a6b8491e05e7efa4e94fd5b6b8490